setup.py

```diff
@@ -15,6 +15,7 @@
     "pygame==2.1.0",
     "shapely==1.8.0",
     "svgpath2mpl==1.0.0",
+    "Jinja2<3.1",

```

Comment:
[readthedocs#9038](https://github.com/readthedocs/readthedocs.org/issues/9038#issue-1180421292)

Reply:
As this is my first commitment to the other's repository, I don't exactly understand what you requested. Please understand me for being clumsy.

Reply:
Hi @MNCHr, thanks for contributing! 

Could you try adding the code below to ``docs/requirements.txt``? I believe ``readthedocs`` uses these instead of the dependencies stored in ``setup.py``.

````
# Ref: https://github.com/readthedocs/readthedocs.org/issues/9038
jinja2<3.1
````

Reply:
Could you also delete the changes to ``setup.py`` before we merge the changes?

Again, thanks for the effort!

Reply:
Hi @stwerner97 ! 
Thank you for your approval and great works !